RecoTracker/TkSeedingLayers/python/DisplacedGeneralLayerTriplet_cfi.py

```diff
@@ -0,0 +1,130 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoTracker.TkSeedingLayers.seedingLayersEDProducer_cfi import *
+
+
+DisplacedGeneralLayerTriplet = seedingLayersEDProducer.clone()
+
+
+DisplacedGeneralLayerTriplet.layerList = cms.vstring(    
```

Comment:
```suggestion
DisplacedGeneralLayerTriplet = seedingLayersEDProducer.clone(


  layerList = [    
```
combine for inline parameter definition